source/data/CategoryList.h

```diff
@@ -13,8 +13,8 @@ You should have received a copy of the GNU General Public License along with
 this program. If not, see <https://www.gnu.org/licenses/>.
 */
 
-#ifndef CATEGORY_LIST_H_
-#define CATEGORY_LIST_H_
+#ifndef DATA_CATEGORY_LIST_H_
+#define DATA_CATEGORY_LIST_H_
```

Comment:
> In the [C](https://en.wikipedia.org/wiki/C_(programming_language)) and [C++](https://en.wikipedia.org/wiki/C%2B%2B) programming languages, #pragma once is a non-standard but widely supported [preprocessor directive](https://en.wikipedia.org/wiki/C_preprocessor) designed to cause the current header file to be included only once in a single compilation.

Since it's non-standard, do we really want to use this?

I mean, as far as I know, both GCC and Clang support it, and they seem to be using it extensively within their own libraries. From what I've heard, MSVC also supports it, though I can't test that. I'm also not filled with confidence by Wikipedia's "its implementation is tricky and might not always be reliable" comment.

Reply:
This is now solved in the latest master-branch.